src/commands/deleteStaticWebApp/IDeleteWizardContext.ts

```diff
@@ -3,8 +3,7 @@
 *  Licensed under the MIT License. See License.txt in the project root for license information.
 *--------------------------------------------------------------------------------------------*/
 
-import { ISubscriptionContext } from "@microsoft/vscode-azext-dev";
-import { ExecuteActivityContext, IActionContext } from "@microsoft/vscode-azext-utils";
+import { ExecuteActivityContext, IActionContext, ISubscriptionContext } from "@microsoft/vscode-azext-utils";
```

Comment:
Both of these imports could just be `import type` though it's not that important.